src/components/portfolio/PortfolioCard.tsx

```diff
@@ -0,0 +1,350 @@
+  ${tw`flex flex-col items-center justify-center`}
+  padding: 8px 16px;
+  background: ${FEE_TIER_BG_COLOR};
+  color: ${FEE_TIER_TEXT_COLOR};
+  border-radius: 100px;
+`;
+
+const BodySubContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: calc(50% - 0.5px);
+  padding-left: 40px;
+  padding-right: 48px;
+  height: 88px;
+  @media (max-width: ${RESPONSIVE_BREAKPOINT_MEDIUM}) {
+    padding: 20px 32px;
+    height: auto;
+  }
+  @media (max-width: ${RESPONSIVE_BREAKPOINT_SMALL}) {
+    width: 100%;
+  }
+`;
+
+const InvestedTypesContainer = styled.div`
+  ${tw`flex flex-col`}
+  gap: 8px;
+`;
+
+const InvestedType = styled.div`
+  display: flex;
+  align-items: center;
+  padding-left: 24px;
+  position: relative;
+  &::before {
+    content: '';
+    position: absolute;
+    left: 0px;
+    top: calc(50% - 4px);
+    width: 8px;
+    height: 8px;
+    border-radius: 100%;
+    background: ${TOKEN_PAIR_FIGURE_COLOR};
+  }
+  &:first-child::after {
+    content: '';
+    position: absolute;
+    left: 3px;
+    top: 16px;
+    width: 2px;
+    height: 24px;
+    background: ${TOKEN_PAIR_FIGURE_COLOR};
+  }
+`;
+
+const DashedDivider = styled.div`
+  margin-left: 8px;
+  margin-right: 8px;
+  position: relative;
+  flex-grow: 1;
+  &::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    top: calc(50% - 1px);
+    width: 100%;
+    height: 1px;
+    border-bottom: 1px dashed ${DASHED_DIVIDER_BORDER_COLOR};
+  }
+`;
+
+const BodyDivider = styled.div`
+  width: 1px;
+  height: 88px;
+  background-color: ${BODY_DIVIDER_BG_COLOR};
+  @media (max-width: ${RESPONSIVE_BREAKPOINT_SMALL}) {
+    width: 100%;
+    height: 1px;
+  }
+`;
+
+export const ValueText = styled.span`
+  font-size: 32px;
+`;
+
+const PercentChangeContainer = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  height: 28px;
+  border-radius: 8px;
+  padding: 6px;
+  font-size: 12px;
+  &:after {
+    content: '';
+    width: 14px;
+    height: 14px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+`;
+
+export const PositivePercentChangeContainer = styled(PercentChangeContainer)`
+  background: ${POSITIVE_PERCENT_BG_COLOR};
+  color: ${POSITIVE_PERCENT_TEXT_COLOR};
+  &:after {
+    background-image: url(${PositiveChangeIcon});
+  }
+`;
+
+export const NegativePercentChangeContainer = styled(PercentChangeContainer)`
+  background: ${NEGATIVE_PERCENT_BG_COLOR};
+  color: ${NEGATIVE_PERCENT_TEXT_COLOR};
+  &:after {
+    background-image: url(${NegativeChangeIcon});
+  }
+`;
+
+// My simple solution to determining the prominent color of the icon
+// (Uses external library)
+// Feel free to replace this with something more robust if need be
+const getProminentColor = async (path: string) => {
+  const values = await prominent(path, { amount: 1 });
+  return `${values[0]}, ${values[1]}, ${values[2]}`;
+};
```

Comment:
This is a good approach for now, we may want to encode some additional behavior for the close-to-white and close-to-black case later.

Reply:
Sounds good!